structs.go

```diff
@@ -358,6 +382,18 @@ type Channel struct {
 
 	// Emoji to use as the default reaction to a forum post.
 	DefaultReactionEmoji ForumDefaultReaction `json:"default_reaction_emoji"`
+
+	// The initial RateLimitPerUser to set on newly created threads in a channel.
+	// This field is copied to the thread at creation time and does not live update.
+	DefaultRateLimitPerUser int `json:"default_rate_limit_per_user"`
```

Comment:
As far as I'm aware, and according to the docs, this field does not exist.
```suggestion
```

Reply:
Sorry, I mistyped the field name, it should have been "default_thread_rate_limit_per_user"